### PR TITLE
Kubernetes demo

### PR DESCRIPTION
**Pull request by devadvocado**
### Full changelog

* [Implement ...]
* [Fix ...]

### Issues resolved

Fix #XXX

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,729 @@
...
+
+### 26. Enable Prometheus metrics on the `mesh` object
+
+```
+$ cat <<EOF | kubectl apply -f - 
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+  namespace: kuma-system
```
Comment (yskopets) [inline kubernetes/README.md]: Let's remove `namespace: kuma-system`.

It's not respected anymore.

File: kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+```
+
+### 17. Let's add logging for traffic between all services and send them to logstash: 
+```
+$ cat <<EOF | kubectl apply -f - 
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+  namespace: kuma-system
```
Comment (yskopets) [inline kubernetes/README.md]: Let's remove `namespace: kuma-system`.

It's not respected anymore.

File: kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,729 @@
...
+  mtls:
+    ca:
+      builtin: {}
+    enabled: true
+  metrics:
+    prometheus: {}
+EOF
+```
+
+### 27. Delete all existing pods in `kuma-demo` so it restarts
```
[the remark this answers is not in the document]
Comment (devadvocado) [reply kubernetes/README.md]: the pods need to restart in order for it to work, am I correct @jakubdyszkiewicz?

[the remark this answers is not in the document]
Comment (jakubdyszkiewicz) [reply kubernetes/README.md]: Unfortunately, the user has to remove Pod now, because we need to add prometheus annotations.

**Pull request merged**